web/wp-content/themes/dctx/analyticstracking.js

```diff
@@ -0,0 +1,7 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
:warning: **Warning**: Equals sign not aligned with surrounding assignments; expected 1 space but found 0 spaces (*Generic.Formatting.MultipleStatementAlignment.NotSameWarning*).